### PR TITLE
Appdev 8529 setup jitterbug vagrant

### PR DESCRIPTION
**Pull request by sharonluong**
This code enables the use of a Homestead VM for Jitterbug development.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,7 +1,73 @@
...
+$ git clone https://github.com/UNC-Libraries/jitterbug.git
+```
+2. `$jitterbug-home` is the directory where you've cloned the jitterbug project. Navigate to it.
+```bash
+$ cd $jitterbug-home
+```
+3. Install composer by following directions at https://getcomposer.org/download/
+
+4. Set up the MySQL Connector.
+    1. Get the MySQL connector [file](http://www.mirrorservice.org/sites/ftp.mysql.com/Downloads/Connector-J/mysql-connector-java-8.0.15.zip) and unzip it.
```
Comment (cazzerson) [inline README.md]: I wonder if this step could be added to `after.sh`

Comment (sharonluong) [reply README.md]: If the server has access to `wget` and `unzip` i probably could? Emily suggested using `wget` and i started writing up instructions for it, but then wasn't sure if it was expected that everyone had those commands in their terminal.

**Comment by sharonluong** [comment]
@cazzerson I added the connector file bit to `after.sh`. Take another look?

**Comment by cazzerson** [comment]
Do you think we could reuse downloaded .zip/jar files across provisioning events? Would speed things up if we didn't have to remove and redownload those files that should never change.

**Comment by sharonluong** [comment]
@cazzerson Yes, I think so. Is there a way to check whether the file exists, and if it doesn't, download it?

**Comment by cazzerson** [comment]
I believe the `--no-clobber` option for wget will only download the file if it doesn't already exist locally.

**Comment by sharonluong** [comment]
@cazzerson it works! `jitterbug-test: File ‘mysql-connector-java-8.0.15.zip’ already there; not retrieving.` Pushing it up now.

File: after.sh [hunk 2]
```diff
@@ -19,15 +19,9 @@ sudo tar xzf solr-7.2.1.tgz solr-7.2.1/bin/install_solr_service.sh --strip-compo
...
+# Get the MySQL connector file and unzip it if needed
 cd /vagrant
-rm -f mysql-connector-java-8.0.15.zip
-cd /opt/solr/contrib/dataimporthandler-extras/lib/
-rm -f mysql-connector-java-8.0.15.jar
-
-# Get the MySQL connector file and unzip it
-cd /vagrant
-sudo wget http://www.mirrorservice.org/sites/ftp.mysql.com/Downloads/Connector-J/mysql-connector-java-8.0.15.zip
+sudo -nc wget http://www.mirrorservice.org/sites/ftp.mysql.com/Downloads/Connector-J/mysql-connector-java-8.0.15.zip
```
Comment (cazzerson) [inline after.sh]: Are the `wget` and `-nc` transposed here?

Comment (sharonluong) [reply after.sh]: 🤦‍♂️

**Comment by cazzerson** [comment]
Looks great--thanks!

**Pull request merged**